src/SAWScript/Interpreter.hs

```diff
@@ -3180,6 +3180,15 @@ primitives = Map.fromList
     , "the live variables in the loop evolve as the loop computes."
     ]
 
+  , prim "llvm_verify_x86_with_invariant"
+    "LLVMModule -> String -> String -> [(String, Int)] -> Bool -> (String, Int, Term) -> LLVMSetup () -> ProofScript () -> TopLevel LLVMSpec"
```

Comment:
What is the significance of the `(String, Int, Term)` value? Explain in the help text.